src/tracks/playabletrack/wavetrack/ui/SpectrumCache.cpp

```diff
@@ -476,7 +476,7 @@ bool WaveClipSpectrumCache::GetSpectrogram(const WaveClip &clip,
    t0 += clip.GetTrimLeft();
 
    const auto track =
```

Comment:
`const auto sequence =`

Reply:
I disagree.  The Sequence object is here cast to a WaveTrack and the Track interface is still really needed.

Reply:
Right. I must have missed the cast somehow.

Reply:
Right. I must have missed the cast somehow.